apps/api/src/app/user/usecases/update-name-and-profile-picture/update-name-and-profile-picture.usecase.ts

```diff
@@ -42,5 +43,10 @@ export class UpdateNameAndProfilePicture {
         _id: command.userId,
       }),
     });
+
+    user = await this.userRepository.findById(command.userId);
+    if (!user) throw new NotFoundException('User not found');
+
+    return this.mapToDto(user);
```

Comment:
return the user data, so on the FE we don't need to refetch it

Reply:
This is a similar approach we use for the topics; but it's a general pattern when you need to transfer data between different layers of the application, in this case, the controller and use-case layers.